bundles/binding/org.openhab.binding.zwave/database/remotec/zts110.xml

```diff
@@ -0,0 +1,143 @@
+			<Type>int</Type>
+			<Default>500</Default>
+			<Minimum>500</Minimum>
+			<Maximum>4000</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Set Filter Counter</Label>
+			<Help lang="en"><![CDATA[Set filter replacement timer from 500 to 4000 hours.]]> </Help>
+		</Parameter>
+	    <Parameter>
+			<Index>4</Index>
+			<Type>int</Type>
+			
+			<Minimum>0</Minimum>
+			<Maximum>9999</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Report Filter Counter</Label>
+			<Help lang="en"><![CDATA[READ ONLY: Data in hours]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>5</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Scale of Temperature</Label>
+			<Help lang="en"><![CDATA[Set Temperature Scale to Farhenheit or Centigrade. DEFAULT =1 Farhenheit]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>6</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Upper Limit of Setpoint A</Label>
+			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>7</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Lower Limit of Setpoint B</Label>
+			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>8</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Easy Mode</Label>
+			<Help lang="en"><![CDATA[Set Easy Mode 0:DISABLE 1:ENABLED DEFAULT:ENABLED ]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>9</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Time Format</Label>
+			<Help lang="en"><![CDATA[Set Easy Mode 0: 24 Hours 1: 12 Hours DEFAULT: 12 Hours ]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>10</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Repeat Basic Set Counter</Label>
+			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
+		</Parameter>
+	</Configuration>
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>4</Maximum>
+			<Label lang="en">Heat Pump</Label>
+			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
+		</Group>
+		<Group>
+			<Index>2</Index>
+			<Maximum>4</Maximum>
+			<Label lang="en">Compressor</Label>
+			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
```

Comment:
What was throwing me was the table on page 31 showing different cases. I
didn't know if that had to be configured in the code as something the user
could define in the binding or if its just a representation of how the
device will work.
On Jul 2, 2015 2:38 PM, "Chris Jackson" <notifications@github.com> wrote:

> In bundles/binding/org.openhab.binding.zwave/database/remotec/zts110.xml
> <https://github.com/openhab/openhab/pull/2838#discussion_r33814668>:
>
> > +			<Label lang="en">Repeat Basic Set Counter</Label>
> > +			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
> > +		</Parameter>
> > +	</Configuration>
> > +	<Associations>
> > +		<Group>
> > +			<Index>1</Index>
> > +			<Maximum>4</Maximum>
> > +			<Label lang="en">Heat Pump</Label>
> > +			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
> > +		</Group>
> > +		<Group>
> > +			<Index>2</Index>
> > +			<Maximum>4</Maximum>
> > +			<Label lang="en">Compressor</Label>
> > +			<Help lang="en"><![CDATA[NOTE THIS PARAMETER NEEDS TO BE UPDATED I DONT KNOW THE PROPER WAY TO WRITE IT. ]]> </Help>
>
> Sorry - I take it back - there are 3 - I was looking at the 100, not the
> 110 version.
> Still, there's nothing to add to the associations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/2838/files#r33814668>.
>